app/presenters/bill-runs/two-part-tariff/remove-bill-run-licence.presenter.js

```diff
@@ -0,0 +1,46 @@
+'use strict'
+
+/**
+ * Formats the data ready for presenting in the remove bill run licence confirmation page
+ * @module RemoveBillRunLicencePresenter
+ */
+
+const { formatFinancialYear } = require('../../base.presenter.js')
+
+/**
+ * Prepares and processes bill run data for presentation
+ *
+ * @param {module:BillRunModel} billRun - an instance of `BillRunModel`
+ * @param {Object[]} licence - an array containing the licence id, ref & account number for a single licence. For the
+ * majority of licences the array will normally contain only one element. However, licences can have multiple billing
+ * accounts so more than one element can exist for a licence.
+ *
+ * @returns {Object} - the prepared data to be passed to the remove licence template
+ */
+function go (billRun, licence) {
```

Comment:
It can be an array, but the array will only contain data for a single licence. There will only be multiple elements if a licence has more than one billing account. So I would say calling it licence is correct.

Reply:
It is not an issue now that we are not displaying the billing account number